pages/common/cargo-remove.md

```diff
@@ -0,0 +1,36 @@
+# cargo remove
+
+> Remove dependencies from a `Cargo.toml` manifest file.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-remove.html>.
+
+- Remove one or more dependencies from the `Cargo.toml` manifest:
+
+`cargo remove`
```

Comment:
I think we can't run bare `cargo remove` . We need to supply the dependency we want to remove as well.
```suggestion
`cargo remove {{dependency_name}}`
```